<launch>
	<!-- Determine the robot for which controllers are to be launched -->
	<arg name="name_space" default="modrob0"/>
	<arg name="controller_description_file" default="$(find modrob_simulation)/config/modrob0_control.yaml"/>
	<!-- Load joint controller configurations from config.yaml to parameter-server -->
	<rosparam file="$(arg controller_description_file)" command="load"/>

	<!-- Loading the joint_position_controllers -->
	<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="$(arg name_space)" args="{eval get_controller_names.py}"/>
	<!-- Loading the motion publisher for this robot -->
	<node name="motion0_publisher" pkg="modrob_simulation" type="motion_publisher" args="0"/>
</launch>